src/main/java/uk/gov/homeoffice/digital/sas/person/listeners/PersonKafkaEntityListener.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.homeoffice.digital.sas.person.listeners;
+
+import jakarta.persistence.PostPersist;
+import jakarta.persistence.PostRemove;
+import jakarta.persistence.PostUpdate;
+import org.springframework.stereotype.Component;
+import uk.gov.homeoffice.digital.sas.kafka.listener.KafkaEntityListener;
+import uk.gov.homeoffice.digital.sas.kafka.producer.KafkaProducerService;
+import uk.gov.homeoffice.digital.sas.person.model.Person;
+
+@Component
+public class PersonKafkaEntityListener extends KafkaEntityListener<Person> {
+
+  public PersonKafkaEntityListener(KafkaProducerService<Person> kafkaProducerService) {
+    super(kafkaProducerService);
+  }
+
+  @Override
+  public String resolveMessageKey(Person person) {
+    return person.getTenantId() + ":" + person.getId();
+  }
+
+  @PostPersist
+  void sendMessageOnCreate(Person resource) {
```

Comment:
I wasn't expecting to see these methods being required, are they not inherited from the base class.
I thought only the resolveMessageKey would be abstract and need an implementation

Reply:
This is being addressed separately: https://github.com/UKHomeOffice/callisto-jparest/pull/58